src/Converter/ConverterInterface.php

```diff
@@ -17,4 +17,18 @@ public function convert(ElementInterface $element);
      * @return string[]
      */
     public function getSupportedTags();
+
+    /**
+     * @param \League\HTMLToMarkdown\ElementInterface $element
+     *
+     * @return void
+     */
+    public function openElement(ElementInterface $element);
+
+    /**
+     * @param \League\HTMLToMarkdown\ElementInterface $element
+     *
+     * @return void
+     */
+    public function closeElement(ElementInterface $element);
```

Comment:
Adding new interface methods is a breaking change that will require a major version bump.  Perhaps we could avoid this by defining a separate interface with these two methods?

Or, if we go with an event dispatcher, maybe we use a generic event listener interface instead?